packages/integration-tests/cmr.js

```diff
@@ -287,27 +319,34 @@ async function generateAndStoreCmrUmmJson(granule, collection, bucket, additiona
  * given S3 location
  *
  * @param {Array<Object>} granules - list of granules in the format of the sync-granules
- * output
+ *                                   output
  * @param {Object} collection - collection object that includes name and version
  * @param {string} bucket - location to save the xmls to
- * @param {string} cmrFileType - CMR file type to generate. Options are echo10, umm_json_v1_4, default
- * is echo10
+ * @param {string} cmrFileType - CMR file type to generate. Options are echo10, umm_json_v1_4,
```

Comment:
`umm_json_v1_5` is another option, correct?

Reply:
yup
and technically probably umm_json_v1_x until they change the MetadataSpecification key.